blog/_posts/2021-01-19-scala-native-0.4-release.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: blog-detail
+post-type: blog
+by: Wojciech Mazur, Scala Center
+title: A sneak peek into the Scala Native 0.4 release
```

Comment:
"Sneak peek" doesn't seem like an appropriate phrase to me here. It normally refers to something that isn't done yet or isn't out yet in its entirety, but you give a little glimpse of some part of it. But the 0.4 release is out.